cmd/security-proxy-setup/res/configuration.toml

```diff
@@ -17,6 +17,7 @@
 # This is a TOML config file for edgexsecurity service.
 
 LogLevel = "DEBUG"
+SNIS = [""]
```

Comment:
Keep for current code, but we should discuss in Security WG if TLS code need to be removed.

Reply:
ok